include/IECorePython/Wrapper.h

```diff
@@ -1,96 +0,0 @@
-//////////////////////////////////////////////////////////////////////////
```

Comment:
Minor thing, but the commit message should say "Use RefCounted*Wrapper* and RunTimeTyped*Wrapper* instead"